pkg/skaffold/deploy/kubectl/labels.go

```diff
@@ -46,7 +46,7 @@ func newLabelsSetter(labels map[string]string) *labelsSetter {
 	}
 }
 
-func (r *labelsSetter) Visit(o map[interface{}]interface{}, k interface{}, v interface{}) bool {
+func (r *labelsSetter) Visit(o map[string]interface{}, k string, v interface{}) bool {
```

Comment:
This is strange. We used to have map[string]interface{} and we fixed something

Reply:
i think this the PR which made `string` to `interface{}` change. https://github.com/GoogleContainerTools/skaffold/pull/2982, 

I added the testcase for it.

Reply:
https://github.com/GoogleContainerTools/skaffold/pull/4201/commits/b90cf4acb94e2b2ee248f928c04be389b1f1c030